x-pack/plugins/serverless_search/public/application/components/connectors/elastic_managed_connector_coming_soon.tsx

```diff
@@ -0,0 +1,367 @@
+/*
```

Comment:
Both this component and the elastic managed web crawlers component have a lot of stuff in common. Instead of duplicating tons of code, we should find ways to re-use some.